docs/config.toml

```diff
@@ -4,7 +4,7 @@ title = "Oso Documentation"
 theme = ["oso-tailwind", "oso-webpack"]
 uglyurls = true
 
-defaultContentLanguage = "any"
+defaultContentLanguage = "python"
```

Comment:
This is a fantastic idea. Kudos to whoever suggested it

Reply:
@gneray

Reply:
Does this work? This has the potential to vastly simplify things

Reply:
It does

Reply:
https://docs-preview.oso.dev/v/dhatch-lang-selector-v2

Reply:
Other than the above comment which is breaking things